src/cgnslib.c

```diff
@@ -5084,12 +5085,12 @@ int cg_poly_elements_general_read(int file_number, int B, int Z, int S,
             s_start, s_end, s_stride,
             section->connect->data_type,
             1, m_dim, m_start, m_end, m_stride, conv_data)) {
-            free(conv_data);
+            if (conv_data) free(conv_data);
```

Comment:
why do we need to check conv_data ?
I assume it was ok because of line 5080

Reply:
fixed, I also removed OSX CI testing.